sdk/contracts.ts

```diff
@@ -83,7 +83,7 @@ const CanonicalAddresses: Record<
     "1.0.0": "0x03B5eBD2CB2e3339E93774A1Eb7c8634B8C393A9",
   },
   [KnownContracts.DELAY]: {
-    "1.0.0": "0xD62129BF40CD1694b3d9D9847367783a1A4d5cB4",
```

Comment:
Should it be a new version number?

Reply:
I went ahead and published the latest version as v1.0.1

Reply:
Sounds good 👌

Reply:
But not on this PR tho, right?

Reply:
I think we should leave the previous version as 1.0.0 and follow the release versioning on each of the module repos from here on out.

Reply:
Right, I for the other modules I remember we use latest, and we only store one version of the ABI. I think accommodating this, is a bigger PR